pkg/annotations/annotations.go

```diff
@@ -4,9 +4,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// Created is a string that can be used as an annotation
-const Created string = "created"
```

Comment:
Was this never used?

Reply:
It used to be, the first annotation was just `created` but it got updated to be the instance identifier